### PR TITLE
fix: add managed account table for supporting Chrome extension to auto login

### PR DESCRIPTION
**Pull request by Resulte**
Screen Shot 2022-08-21 at 21 48 49

https://user-images.githubusercontent.com/34178764/185437883-fdae944e-abe3-4abf-b493-8319b59174a3.mov

### CODE REVIEW

File: web/src/locales/zh/data.json [hunk 1]
```diff
@@ -636,6 +636,7 @@
     "Link": "绑定",
     "Location": "城市",
     "Location - Tooltip": "居住地址所在的城市",
+    "Manage accounts": "Manage accounts",
```
Comment (hsluoyz) [inline web/src/locales/zh/data.json:639]: zh should be translated

File: web/src/ManageAccountTable.js [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+        },
+      },
+      {
+        title: i18next.t("general:Action"),
+        key: "action",
+        width: "50px",
+        render: (text, record, index) => {
+          return (
+            <div>
+              <Tooltip placement="topLeft" title={i18next.t("general:Delete")}>
```
Comment (hsluoyz) [inline web/src/ManageAccountTable.js:113]: Keep the up and down buttons

**Review by hsluoyz — CHANGES_REQUESTED** [review]

File: web/src/ManagedAccountsTable.js [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+
+import React from "react";
+import {DeleteOutlined, DownOutlined, UpOutlined} from "@ant-design/icons";
+import {Button, Col, Input, Row, Select, Table, Tooltip} from "antd";
+import * as Setting from "./Setting";
+import i18next from "i18next";
+
+const {Option} = Select;
+
+class ManagedAccountsTable extends React.Component {
```
Comment (hsluoyz) [inline web/src/ManagedAccountsTable.js:23]: ManagedAccountsTable -> ManagedAccountTable

File: web/src/UserEditPage.js [hunk 4]
```diff
@@ -554,6 +555,22 @@ class UserEditPage extends React.Component {
           </Col>
         </Row>
       );
+    } else if(accountItem.name === "Managed accounts") {
```
Comment (hsluoyz) [inline web/src/UserEditPage.js:558]: space after "else if"

**Review by hsluoyz — CHANGES_REQUESTED** [review]

File: object/user.go [hunk 5]
```diff
@@ -114,6 +114,8 @@ type User struct {
 
 	LastSigninWrongTime string `xorm:"varchar(100)" json:"lastSigninWrongTime"`
 	SigninWrongTimes    int    `json:"signinWrongTimes"`
+
+	ManagedAccounts []ManagedAccount `xorm:"managedAccounts blob" json:"managedAccounts"`
```
Comment (hsluoyz) [inline object/user.go:118]: why not use `varchar(xxx)`?

File: object/organization.go [hunk 6]
```diff
@@ -46,7 +46,7 @@ type Organization struct {
 	EnableSoftDeletion bool     `json:"enableSoftDeletion"`
 	IsProfilePublic    bool     `json:"isProfilePublic"`
 
-	AccountItems []*AccountItem `xorm:"varchar(2000)" json:"accountItems"`
+	AccountItems []*AccountItem `xorm:"varchar(3000)" json:"accountItems"`
```
Comment (hsluoyz) [inline object/organization.go:49]: why change this?

File: web/src/UserEditPage.js [hunk 7]
```diff
@@ -31,6 +31,7 @@ import WebAuthnCredentialTable from "./WebauthnCredentialTable";
 
 import {Controlled as CodeMirror} from "react-codemirror2";
 import "codemirror/lib/codemirror.css";
+import ManagedAccountsTable from "./ManagedAccountsTable";
```
Comment (hsluoyz) [inline web/src/UserEditPage.js:34]: put this after line 30

**Review by hsluoyz — CHANGES_REQUESTED** [review]

File: object/user.go (hunk 5 above)
Comment (Resulte) [reply object/user.go:118]: I use the same struct as WebauthnCredentials `WebauthnCredentials []webauthn.Credential `xorm:"webauthnCredentials blob" json:"webauthnCredentials"``

File: object/organization.go (hunk 6 above)
Comment (Resulte) [reply object/organization.go:49]: Because when I store too many of AccountItems, the database will report an out-of-memory error.

Comment (hsluoyz) [reply object/organization.go:49]: OK

File: object/user.go (hunk 5 above)
Comment (hsluoyz) [reply object/user.go:118]: OK

**Review by hsluoyz — APPROVED** [review]

**Pull request merged**